python/lsst/ts/atbuilding/csc/building_csc.py

```diff
@@ -0,0 +1,378 @@
+# This file is part of ts_atbuilding_csc
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["ATBuildingCsc", "run_atbuilding"]
+
+import asyncio
+import json
+from collections import defaultdict
+from typing import Any
+
+from lsst.ts import salobj, tcpip, utils
+from lsst.ts.xml.enums.ATBuilding import FanDriveState, VentGateState
+
+from . import __version__
+from .config_schema import CONFIG_SCHEMA
+from .enums import ErrorCode
+from .mock_controller import MockVentController
+
+RPI_AVAILABLE = True
+try:
+    from lsst.ts.vent.controller import Config, Controller, Dispatcher
+except ImportError:
+    RPI_AVAILABLE = False
+
+# Max time (sec) to wait for the mock controller to start.
+MOCK_CTRL_START_TIMEOUT = 2
+
+# Max time (sec) to wait for a TCP/IP command to complete.
+TCP_TIMEOUT = 1
+
+# Max time (sec) to receive a message from the server.
+# This might be a while if no commands are active.
+SERVER_MESSAGE_TIMEOUT = 30
+
+
+class ATBuildingCsc(salobj.ConfigurableCsc):
+    """AuxTel Building CSC (dome vents and fan)
+
+    Parameters
+    ----------
+    initial_state : `salobj.State` or `int` (optional)
+        The initial state of the CSC. This is provided for unit testing,
+        as real CSCs should start up in `lsst.ts.salobj.StateSTANDBY`,
+        the default.
+    simulation_mode : `int` (optional)
+        Simulation mode.
+
+    Raises
+    ------
+    salobj.ExpectedError
+        If initial_state or simulation_mode is invalid.
+
+    Notes
+    -----
+    **Simulation Modes**
+
+    Supported simulation modes
+
+    * 0: regular operation
```

Comment:
Why to you need 2 options for simulation mode? can't you get away with only one?